maps/submaps/surface_submaps/wilderness/spider1.dmm

```diff
@@ -3,18 +3,19 @@
 "c" = (/turf/template_noop,/area/submap/spider1)
 "d" = (/obj/structure/flora/tree/pine,/turf/template_noop,/area/submap/spider1)
 "e" = (/obj/effect/spider/eggcluster/small/frost,/turf/template_noop,/area/submap/spider1)
-"f" = (/mob/living/simple_animal/hostile/giant_spider/frost,/turf/template_noop,/area/submap/spider1)
-"g" = (/obj/structure/flora/tree/sif,/turf/template_noop,/area/submap/spider1)
-"h" = (/obj/effect/spider/cocoon,/mob/living/simple_animal/hostile/giant_spider/frost,/turf/template_noop,/area/submap/spider1)
+"f" = (/mob/living/simple_animal/hostile/giant_spider/frost{returns_home = 1},/turf/template_noop,/area/submap/spider1)
+"g" = (/mob/living/simple_animal/hostile/giant_spider/nurse,/turf/template_noop,/area/submap/spider1)
```

Comment:
Should that spider be set to return home like the frost spiders are?

Reply:
It was specifically set not to. It's a single nurse, the hope is that it'll wander around and make webs, so the PoI isn't two spiders and a bunch of web surrounded by a million miles of nothing else.

And, again, single nurse. Not really threatening in any way.